internal/pool/pool_single_test.go

```diff
@@ -0,0 +1,31 @@
+package pool_test
+
+import (
+	"context"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/go-pg/pg/v10/internal/pool"
+)
+
+var _ = Describe("SingleConnPool", func() {
+	var p *pool.SingleConnPool
+
+	BeforeEach(func() {
+		p = pool.NewSingleConnPool(nil, &pool.Conn{})
+	})
```

Comment:
Would this need to be a before each and a variable in this scope? Or could the pool just get moved into the test itself since there is only one?